packages/bot-web-ui/src/app/app.jsx

```diff
@@ -3,32 +3,25 @@ import React from 'react'; // eslint-disable-line import/first
 import { Loading } from '@deriv/components';
 import { DBot, ServerTime, ApiHelpers } from '@deriv/bot-skeleton'; // eslint-disable-line import/first
 import {
-    Audio,
-    BlocklyLoading,
-    BotFooterExtensions,
+    Dashboard,
     BotNotificationMessages,
-    MainContent,
+    NetworkToastPopup,
     QuickStrategy,
-    RunPanel,
+    Audio,
     RoutePromptDialog,
-    Toolbar,
-    NetworkToastPopup,
+    BotFooterExtensions,
 } from 'Components';
```

Comment:
can we import the components in alphabetical order so it's easier to read?

Reply:
done ✅